### DISCUSSION TITLE
Allow for passwordless unattended access

### DISCUSSION THREAD

**Discussion by stavros-k** [rustdesk/rustdesk#7676] [General]
I would like to request a way to connect unattended to devices added to the console, without having to set password on the address book and on the remote computer.

With the same way the settings can be synced from the server to the remote computer, also sync a random generated password per remote computer (which also the address book will automatically use).

We want to add ~1500 unattended clients (kiosks). Its very tedious to manually set password to 1500 clients in the address book.
and will also be pain to roll those passwords every couple of weeks/months

**Comment by rustdesk** [comment]
https://www.reddit.com/r/rustdesk/comments/1bqg7p8/device_password_management/
